js/securesubmit/checkout-form.js

```diff
@@ -857,17 +859,23 @@ document.observe('dom:loaded', function () {
                 });
 
                 if (window.secureSubmitAmastyCompleteCheckoutOriginal && document.getElementById('amscheckout-onepage')) {
-                    if(document.getElementById('ss-banner'){
-                        document.getElementById('ss-banner').style.backgroundSize = '325px 40px';
+                    
+                    var ssbanner = document.getElementById('ss-banner');
+                    var ccnumber = document.getElementById('cc-number');
+                    var expirationdate = document.getElementById('expiration-dat');
+                    var ccv = document.getElementById('payment-buttons-container');
+                    
+                    if(ssbanner){
+                       ssbanner.style.backgroundSize = '325px 40px';
                     }
-                    if(document.getElementById('cc-number')){
-                        document.getElementById('cc-number').className = 'securesubmit_amasty_one_page_checkout';
+                    if(ccnumber){
+                        ccnumber.className = 'securesubmit_amasty_one_page_checkout';
                     }
-                    if(document.getElementById('expiration-date')){
-                        document.getElementById('expiration-date').className = 'securesubmit_amasty_one_page_checkout';
+                    if(expirationdate){
+                        expirationdate.className = 'securesubmit_amasty_one_page_checkout';
                     }
-                    if(document.getElementById('ccv')){
-                        document.getElementById('ccv').className = 'securesubmit_amasty_one_page_checkout';
+                    if(ccv){
+                        ccv.className = 'securesubmit_amasty_one_page_checkout';
```

Comment:
@ttnnkkrr Again, sorry for not catching this before, but just setting `.className` to a value will overwrite all previous values. With our convenience functions, classes may be added that could potentially be overwritten with these calls.